_posts/2024-05-09-djangocon-us-call-for-venue-proposal-2025.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title:  "DjangoCon US: Call for Venue Proposal 2025"
+date: 2024-05-09 12:00:00
+categories: ""
+author_name : DEFNA
+author_url : /about
+author_avatar: defna
+show_avatar : true
+read_time : ""
+feature_image: 2016-06-08
+show_related_posts: false
+permalink: /announcements/2024/5/9/djangocon-us-call-for-venue-proposal-2025/
+---
+
+We’re in the middle of planning DjangoCon US 2024, so it’s time to think about 2025! Where will we be next year? We need your help in making that decision!
+
+So you want your city to host DjangoCon US? Hurrah! We’re delighted by your interest. Here’s what you need to know, and what we need to know, to consider your proposal.
+
+For 2025, we are looking at conference dates of October 5-10, 2025 or October 12-17, 2025.
+
+We need some information about your city, about the venue, the hotel, and travel. Your proposal doesn’t have to be super-fancy, just informative. The proposal should include the following:
+
+1. Names of organizer staff in the host city. While DEFNA will handle the business aspects of the conference, we need the help of a team of local volunteers to manage preparation and the onsite coordination of the event. We realize you won't have an entire team assembled, but let us know who the core organizers in your city will be.
+
+2. A short list of possible conference venues (or just one, if you have found the Venue of Conference Dreams), with the following information:
+
+	* Name
+	* Location
+	* Transportation options (how do people get there from the airport?)
+	* Attendance capacity
+	* Assurance that the venue can provide the following spaces (A floorplan with a rough explanation of what will go where would be great.):
+		* Keynote room (also Sessions Track A) to accommodate up to 450 people, classroom style (tables with chairs)
+		* Room for Sessions Track B to accommodate up to 200 people, classroom style (tables with chairs)
+		* Space for sponsors to set up exhibit booths/tables; at least 12, and up to 16, sponsors with either 6’ tables or 10’ x 10’ booth spaces. If more sponsor booth space is available, that's a plus.
+		* Green room for speakers to prepare (generally a room that will seat 25 for a workshop is the right size)
+		* Space for registration tables
+		* Room for meals and breaks with sufficient seating and tables. If serving and/or seating areas cannot accommodate 450 people at once, indicate the room capacity and time required to move 450 people through.
```

Comment:
Do we still want to plan for 450 people? Would it make a difference if we lower this to 400?

Reply:
I think we will eventually bounce back to pre-pandemic levels. Most of our venues can easily seat 600 to 800 people. They tend to be <300 or under <800 or 1000. So I'm not sure 50 people are make or break. 

I don't think 300 is sustainable to our goals.